pallets/author-mapping/src/lib.rs

```diff
@@ -57,13 +58,31 @@ pub mod pallet {
 		deposit: Balance,
 	}
 
+	#[derive(Encode, Decode, PartialEq, Eq, Debug, scale_info::TypeInfo)]
+	#[scale_info(skip_type_params(T))]
+	pub struct RegistrationInformation<T: Config> {
+		account: T::AccountId,
+		deposit: BalanceOf<T>,
+		keys: T::Keys,
+	}
+
+	/// need a function to convert from existing AuthorMapping to current AuthorMapping
```

Comment:
I think the best is to add trait bond : `Keys: From<NimbusId>`, to be able to define how to set ""default keys".
In the VRF case, the crypto scheme is the same as nimbus, it's sr25519, so we can create a impl that set the VRF key with the same bytes as the NimbusId.

Reply:
done